src/main/ffUtils.ts

```diff
@@ -2,11 +2,11 @@ import ffmpegStaticPath from 'ffmpeg-static';
 import ffprobeStatic from 'ffprobe-static';
 
 export const ffmpegPath = ffmpegStaticPath.replace(
-  'app.asar',
-  'app.asar.unpacked'
+  'app.asar\\dist\\main\\',
+  'ffmpeg-static\\'
 );
 
 export const ffprobePath = ffprobeStatic.path.replace(
-  'app.asar',
-  'app.asar.unpacked'
+  'app.asar\\dist\\main\\',
+  'ffprobe-static\\'
```

Comment:
It still spins forever on linux (and I suspect it will do the same for macos)

This is probably the cause. You are using `\\` which is only used by windows (wont work on linux/macos).

Could you try:

```javascript
import ffmpegStaticPath from 'ffmpeg-static';
import ffprobeStatic from 'ffprobe-static';
import path from 'path';

export const ffmpegPath = ffmpegStaticPath.replace(
  path.join('app.asar', 'dist', 'main'),
  path.join('ffmpeg-static')
);

export const ffprobePath = ffprobeStatic.path.replace(
  path.join('app.asar', 'dist', 'main'),
  path.join('ffprobe-static')
);
```

Reply:
My bad, even if you try it you wont be able to try linux/macos :laughing: 

I just tested that solution and it worked. No more infinite spinning.